packages/elements/src/components/ino-popover/ino-popover.tsx

```diff
@@ -81,26 +94,41 @@ export class Popover implements ComponentInterface {
     this.create();
   }
 
-  // Private methods
+  /**
+   * Emits when an element which is not part of the popover is clicked.
+   * Should be used if you control the state of the popover.
+   */
+  @Event() clickOutside: EventEmitter<void>;
```

Comment:
Isn't this a bottleneck? Like this we can only use the popover in controlled mode using a click. What happens if I define a hover trigger, f.e?

Reply:
Yea, if we want to be really consistent, there also should be events like `hoverOutside`, ...
In this case, I just forwarded the tippy API (`onClickOutside`) and thought it is nice to have if you want to implement custom logic.

Reply:
https://github.com/inovex/elements/issues/291